src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -354,15 +355,64 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
     artwork.isAcquireable || activePartnerOffer
   )
 
+  const SaleMessageOrOfferDisplay: FC = () => {
+    if (activePartnerOffer) {
```

Comment:
minor: Should this component be defined outside the main component body? Then artwork/activePartnerOffer could be props and the function wouldn't need to be redefined.

Reply:
I just moved the actual display code to OfferDisplay component similar to how SaleMessage is done here. Theoretically we can make it to be stand alone thing just defining it's own fragment but for now did not seem necessary. 

Can refactor separately if we need to reuse it in any other places.